webapp/src/main/webapp/WEB-INF/conf/news/metadata-plugins-configuration.xml

```diff
@@ -69,12 +69,34 @@
               <string>snapshotSliderNews</string>
             </field>
             <field name="properties">
-              <map type="java.util.HashMap">
-                <entry>
-                  <key><string>label</string></key>
-                  <value><string>Snapshot slider news</string></value>
-                </entry>
-              </map>
+              <collection type="java.util.ArrayList">
```

Comment:
to be updated according to social [Social PR](https://github.com/Meeds-io/social/pull/1103)